src/api/app/models/review.rb

```diff
@@ -58,6 +58,7 @@ class NotFoundError < APIError
   scope :bs_request_ids_of_involved_users, ->(user_ids) { where(user_id: user_ids).select(:bs_request_id) }
 
   scope :declined, -> { where(state: :declined) }
+  scope :open, -> { where(state: :new) }
```

Comment:
Hm why do you invent a new term for the scope?

Reply:
"Open reviews" is a term we use more often when talking about this. Moreover, renaming the scope to `new`will make us use it like `Review.new`, what can be even more confusing.

So it's been changed to `in_state_new`.